sn_client/src/wallet.rs

```diff
@@ -201,8 +279,10 @@ impl WalletClient {
     }
 
     /// Send signed spends to another wallet.
-    /// Can optionally verify the store has been successful (this will attempt to GET the Spend from the network)
-    pub async fn send_signed_spends(
```

Comment:
The change is here. The rest of the code comes from a branch that's waiting to merge.